src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -324,7 +324,7 @@ public boolean onCommand(@Nullable CommandSender sender, @Nullable Command comma
 				info(sender, "info.documentation");
 				info(sender, "info.server", Bukkit.getVersion());
 				info(sender, "info.version", Skript.getVersion());
-				info(sender, "info.addons");
+				info(sender, "info.addons", Skript.getAddons().isEmpty() ? "None" : "");
```

Comment:
```suggestion
				info(sender, "info.addons", Skript.getAddons().isEmpty() ? "none" : "");
```

Reply:
I'd argue the uppercase "None" is correct in this case:
In both British English and American English, separate sentences after a segmental use of a colon are capitalised. Also, using `None` would be consistent with plugin names, which are usually in pascal case (with some rare exceptions like `spark`)

Reply:
I think blue have given the official side of the answer, adding to that it just looks better capitalized also following the plugin naming.

Reply:
Agree with @bluelhf changes aren't needed.

Reply:
Okay